README.md

```diff
@@ -38,13 +38,17 @@ ArcticDB is designed from the outset to be resilient; there is no single point o
 
 ## Quickstart
 
-> ⚠️ **Pre-built binaries only available for Linux and Windows**: MacOS binaries are coming soon!
+> ⚠️ **Pre-built binaries are available for Linux, Windows and MacOS (Apple silicon)**
```

Comment:
The situation is more confusing than this isn't it, we have:

PyPi:
- Windows, Linux

Conda:
- Linux, MacOS, (Windows soon)

I don't know whether we want to document that complexity, or just wait till we have MacOS for Windows too.

Reply:
table added